tests/unit/browser/test_webelem.py

```diff
@@ -325,6 +327,38 @@ def test_debug_text(self, elem, xml, expected):
         elem._elem.toOuterXml.return_value = xml
         assert elem.debug_text() == expected
 
+    def test_remove_blank_target(self):
```

Comment:
I think the test is rather big, and testing different things - IMHO it'd make sense to add a `TestRemoveBlankTarget` class (like `TestIsVisible` below) with individual tests.